src/cmds/stack/setup.js

```diff
@@ -53,13 +54,15 @@ const cpDevDir = (basePath, version) => {
 };
 
 const cloneRepositories = (initPath, repositories) => {
-  repositories.forEach((repo) => {
+  repositories.forEach((fullRepo) => {
+    const [repo, checkout] = fullRepo.split(':');
```

Comment:
Could we be more descriptive? Also, `checkout` is what the variable is used for, not what value it's holding :eyes: 

Here's a suggestion :point_down: 
```suggestion
  repositories.forEach((repository) => {
    const [repo, branch] = repository.split(':');
```

Reply:
You're right! I've accepted your suggestions! :smiley: